lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -242,9 +226,18 @@ defmodule PhxToolsWeb.PhxToolsComponents do
     ]
   end
 
+  defp installation_instructions(_os) do
```

Comment:
Unnecessary code duplication.
We can do better with proper pattern matching.
How about just these two functions?

```elixir
defp installation_instructions(:macos) do
  ...
end

defp installation_instructions(_os) do
  ...
end
```